plugins-structure/packages/ticketvote/webpack.common.js

```diff
@@ -0,0 +1,49 @@
+const path = require("path");
+const pkgName = require("./package.json").name;
+
+const jsRules = {
+  test: /\.js?$/,
+  loader: "babel-loader",
+  exclude: /node_modules/,
+  options: {
+    presets: ["@babel/preset-env", "@babel/preset-react"],
+    plugins: ["@babel/plugin-transform-runtime"],
+  },
+};
+
+const cssRules = [
+  {
+    test: /\.css$/,
+    use: [
+      "style-loader",
+      {
+        loader: "css-loader",
+        options: {
+          importLoaders: 1,
+          modules: true,
+        },
+      },
+    ],
+    include: /\.module\.css$/,
+  },
+  {
+    test: /\.css$/,
+    use: ["style-loader", "css-loader"],
+    exclude: /\.module\.css$/,
+  },
+];
+
+module.exports = {
+  entry: "./src/index.js",
+  output: {
+    path: path.resolve(__dirname, "dist"),
+    library: {
+      name: pkgName + ".js",
+      type: "umd",
+    },
+    clean: true,
+  },
```

Comment:
The issue you told me on matrix with deep routing, e.g `/ticketvote/:ticketvoteid` is because of the webpack `publicPath` variable on your webpack env. Just add `publicPath: "/"` to the output here and it will work fine. In sum, your app wasn't finding the compiled `main.js` file. See more at: https://webpack.js.org/guides/public-path/. In the future, pay attention to the core configs.

Reply:
It works! Thanks!

Reply:
https://github.com/tiagoalvesdulce/politeiagui/pull/8 should fix it.